client/src/pages/About/About.module.scss

```diff
@@ -0,0 +1,22 @@
+.community-video {
+    box-sizing: border-box;
+
+    width: 1140px;
+    height: 480px;
+
+    /* Primary */
+
+    background: #5c11d5;
+    /* Divider */
+
+    border: 1px solid rgba(0, 0, 0, 0.12);
```

Comment:
what should I do if there isn't a matching variable? should I touch the scss file?

Reply:
if you do:
@import '@/styles/setup/variables';
you will have all the colors - like primary and divider. we have it.
(I don't know why by mistake the primary color was pushed by me in rgb pattern instead of hex, but it's the same color - I'll fix it).
In  general, if a color doesn't appear in the variables list but appears in Figma color palette with a name, you add the variable. If it doesn't have a name, I would add just the color in the component css file and we can ask the UI team about it.